proposals/drafts/elections.rst

```diff
@@ -43,46 +255,502 @@ extras
     Common to all Open Civic Data types, the value is a key-value store suitable for storing arbitrary information not covered elsewhere.
 
 
+BallotMeasureContest
+--------------------
+
+A subclass of ``ContestBase`` for representing a ballot measure before the voters, including summary statements on each side.
+
+id
+    Open Civic Data-style id in the format ``ocd-ballotmeasurecontest/{{uuid}}``
+
+con_statement
+    **optional**
+    Specifies a statement in opposition to the ballot measure. It does not necessarily appear on the ballot (string).
+
+effect_of_abstain
+    **optional**
+    Specifies the effect abstaining from voting on the ballot measure, i.e., whether abstaining is considered a vote against it (string).
+
+full_text
+    **optional**
+    Specifies the full text of the ballot measure as it appears on the ballot (string).
+
+info_uri
+    **optional**
+    Specifies a URI that links to additional information about the ballot measure (string).
+
+passage_threshold
+    **optional**
+    Specifies the threshold of votes the ballot measure needs in order to pass. The default is a simple majority, i.e., "50% plus one vote". Other common thresholds are "three-fifths" and "two-thirds".
+
+pro_statement
+    **optional**
+    Specifies a statement in favor of the referendum. It does not necessarily appear on the ballot (string).
+
+summary_text
+    **optional**
+    Specifies a short summary of the ballot measure that is on the ballot, below the title, but above the text.
+
+type
+    **optional**
+    Enumerated among "ballot-measure", "referendum", "initiative" and "other".
```

Comment:
Add definitions to each of these enums, which again are all taken from VIP. They could definitely be more specific. For example, is it a constitutional or statutory. Also can't a repeal referendum be an initiated by group of citizens? Take a look at the definitions I've added in the [other PR](https://github.com/opencivicdata/docs.opencivicdata.org/pull/64), and if you think they're still lacking, let's discuss there.